src/VisualStudio/Xaml/Impl/Features/OrganizeImports/XamlOrganizeImportsService.cs

```diff
@@ -28,11 +28,9 @@ public XamlOrganizeImportsService(IXamlOrganizeNamespacesService organizeService
             _organizeService = organizeService;
         }
 
-        public async Task<Document> OrganizeImportsAsync(Document document, CancellationToken cancellationToken)
+        public async Task<Document> OrganizeImportsAsync(Document document, OrganizeImportsOptions options, CancellationToken cancellationToken)
```

Comment:
Will this require a coordinated insertion?

Reply:
We can merge to Roslyn and then when O# is updated to the latest Roslyn it will require https://github.com/OmniSharp/omnisharp-roslyn/pull/2343

Reply:
Sorry, I meant for the XAML tooling.

Reply:
It shouldn't since this is not modifying APIs they use. They implement IXamlOrganizeNamespacesService.

Reply:
I agree with @tmat. As long as you don't change our IXamlXxxx interfaces then everything should be good. This is just a Roslyn implementation change of how to get the options.

Reply:
Ah, I should have checked for it under an API namespace or ExternalAccess project. Thanks